packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/InteractionLayerContainer.js

```diff
@@ -69,13 +73,12 @@ export function InteractionLayerContainer({
   played,
   duration
 }) {
-
   const newMarks =
     shownMarks === SHOWN_MARKS.NONE ? marks.slice(hidingIndex) : marks
-  const visibleMarksPerFrame = newMarks?.filter(
-    (mark) => mark.frame === frame
-  )
-
+  const visibleMarksPerFrame = (multiImageCloneMarkers)
+    ? newMarks
+    : newMarks?.filter((mark) => mark.frame === frame)
+    
```

Comment:
By referencing `multi_image_clone_markers` only in this one spot, we're not meeting the expectation described in [your comment](https://github.com/zooniverse/front-end-monorepo/issues/5493#issuecomment-1852488736) on the linked Issue:
> ...when we clone marks across frames the mark.frameIndex will always be 0 instead of whichever frame the mark was drawn or edited.

Here's an example describing the combo of `multi_image_clone_markers` and `frame`:

I created a test production workflow where the subjects are multi-frame and there are two steps. Each step is one point tool. https://localhost.zooniverse.org:3000/projects/goplayoutside3/fem-flipbook-viewer/classify/workflow/24066?env=production

Run the project locally on this branch, and when `multi_image_clone_markers` is true, the mark I draw in the first step displays on all frames. I drew it on the first frame while testing. However, when I go to the next step, the mark I drew in the first step is no longer cloned and it only displays on the first frame. This is also true for the second step. For example if I'm on the second step, I draw a mark on 3rd frame and it displays as cloned, then go back to the first step, that mark is no longer cloned. It only shows up on the 3rd frame. 

I believe this is happening because when `multi_image_clone_markers` is true, there's no handling of `mark.frame` for _new_ marks in this PR. Try out the above test workflow by drawing marks on different frames, submit the classification and in the logged object, you'll see the `annotations` array contains marks with `frame` as the actual frame I clicked on. According to the comment above, new marks created when `multi_image_clone_markers = true` should all have `frame: 0`. 

Accomplishing this might look something like `interactionFrame = multiImageCloneMarkers ? 0 : frame` in InteractionLayerContainer, and then passing `interactionFrame` to the InteractionLayer for its `createMark()` function. I think you'd also need to pass `interactionFrame` to PreviousMarks.

(feel free to toggle cloning config on my test workflow while in admin mode: https://www.zooniverse.org/lab/19645/workflows/24066).